taf/models/types.py

```diff
@@ -54,6 +54,16 @@ def is_yubikey(self):
             self.yubikey is True or (self.yubikeys is not None and len(self.yubikeys))
         )
 
+    @property
+    def yubikey_ids(self):
+        if not self.is_yubikey:
+            return []
```

Comment:
I believe this should return None, since in `taf/keys.py`  L307, L308 there is a check for None